BlueprintUI/Sources/BlueprintView/ElementPreview.swift

```diff
@@ -67,6 +67,7 @@ public struct ElementPreview: View {
     public typealias ElementProvider = () -> Element
 
     private let name: String
+    private let previewName: String?
```

Comment:
I believe this is only used for previews—I think we might want to just remove the prefix when using `name`.

I think we just want an optional `previewName` override on the Market side.

Thoughts?

Reply:
Okay, here's the result of that suggestion:
![image](https://github.com/square/Blueprint/assets/141675/94f10164-3209-4c12-9284-f396f575ec24)

@kyleve do you have opinions? is keeping the `prefix - name` worthwhile in any cases?